public/site.webmanifest

```
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```

The webmanifest (for creating Progressive Web Applications) should be human-readable and have a few entries filled out:

```suggestion
{
  "name": "Predictive Text Studio",
  "short_name": "PTStudio",
  "icons": [
    {
      "src": "/android-chrome-192x192.png",
      "sizes": "192x192",
      "type": "image/png"
    },
    {
      "src": "/android-chrome-512x512.png",
      "sizes": "512x512",
      "type": "image/png"
    }
  ],
  "theme_color": "#0099ff",
  "background_color": "#ffffff",
  "display": "standalone"
}
```